web-ui/src/main/resources/catalog/components/search/resultsview/partials/viewtemplates/editor.html

```diff
@@ -11,8 +11,15 @@
          title="{{md.abstract}}">{{md.title || md.defaultTitle}}</a>
       <span data-ng-show="md.isTemplate == 's'">{{md.title || md.defaultTitle}}</span>
       <br/>
-      <div class="gn-record-details">
-        <span title="{{'metadataAuthor' | translate}}">{{md.getOwnername()}}</span>
+      <div class="gn-record-details text-muted">
+        <small><span data-translate="">owner</span>:</small>
+        <small>{{::md.getOwnername()}}</small>
+        &nbsp;
+        <small data-translate="">updatedOn</small>
+        <small class="text-muted"
+               data-gn-humanize-time="{{md['geonet:info'].changeDate}}"
+               data-from-now=""></small>
```

Comment:
I guess you could probably put all this in one `<small>` tag?

Reply:
> I guess you could probably put all this in one <small> tag?

Probably not one because of ```data-gn-humanize-time``` so maybe 2. I keep this like this until a better refactor.